src/pushsource/_impl/model/ami.py

```diff
@@ -180,6 +201,13 @@ class AmiPushItem(VMIPushItem):
     )
     """Automatically created security groups for the product. """
 
+    access_endpoint_url = attr.ib(
+        type=AmiAccessEndpointUrl,
+        default=None,
+        validator=optional(instance_of(AmiAccessEndpointUrl)),
+    )
+    """Billing codes associated with this image."""
```

Comment:
Another false copy-pasted comment/doc string (this will be part of the public docs so it has to be fixed).

Reply:
same here